### DISCUSSION TITLE
Implications of choosing 2 App servers

### DISCUSSION THREAD

**Discussion by Andy-Roger-Kats** [mozilla/hubs#3199] [Hubs Cloud Questions] [answered]
Our Hubs Cloud setup is using 2 app servers, I noticed that a room I created will sometimes connect to one and other times connect to the other. Is this supposed to happen? On the surface this would seem to prohibit two users in the same room to share phoenix channel communication if their clients connect to different app servers.

Another related issue / question is I am trying to ssh into the other app server's ec2 instance but I can only ssh into one of them. I am using the provided .pem file (there was only one as far as I can tell). How do I ssh into the other instance -or is this not possible?

**Comment by robinkwilson** [comment, marked as answer]
Yes, nice attention to detail! That's supposed to happen, automatic load balancing happening in the background.

**SSH into each of your two app servers**
In AWS console > EC2 running instances, you'll see your two app servers running. To SSH into each box, select the EC2 instance you'd like to ssh into and can follow the instructions provided by AWS. The ssh key contained in your .pem file should work for both.

This will help too: https://hubs.mozilla.com/docs/hubs-cloud-accessing-servers.html

For example, I use the command line to ssh, I use these commands: (*"aws-named-ec2-1st-box" is the name of your box aws generated, you can see it in the aws console*)
`ssh ubuntu@aws-named-ec2-1st-box.<<internal domain>>`
`ssh ubuntu@aws-named-ec2-2nd-box.<<internal domain>>`

**Comment by vincentfretin** [comment]
@Andy-Roger-Kats You may find some answers in the code, it appears to me that all participants go to the same Phoenix instance, see my comment at https://github.com/mozilla/hubs/discussions/2887#discussioncomment-58885
